UpdateVersions.ps1

```diff
@@ -3,16 +3,23 @@ Param(
     [string]$NuGetPackagesFolder = ""
 )
 
-# If the nuget packages folder is specified containing the latest versions,
-# go through them to get the versions of the component packages.
+# First, add the metapackage
 $nugetPackageToVersionTable = @{"Microsoft.WindowsAppSDK" = $WinAppSDKVersion}
+
+# Next, if the nuget packages folder is specified containing the latest versions,
+# go through them to get the versions of all dependency packages.
 if (!($NuGetPackagesFolder -eq ""))
 {
     Get-ChildItem $NuGetPackagesFolder | 
-    Where-Object { $_.Name -like "Microsoft.WindowsAppSDK*" } | 
+    Where-Object { $_.Name -like "Microsoft.WindowsAppSDK.*" -or 
+                   $_.Name -like "Microsoft.Windows.SDK.BuildTools.*" -or 
+                   $_.Name -like "Microsoft.Web.WebView2.*" } | 
     Where-Object { $_.Name -notlike "*.nupkg" } |
     ForEach-Object { 
-        if ($_.Name -match "^(Microsoft\.WindowsAppSDK\.[a-zA-Z]+)\.([0-9].*)$")
+        if ($_.Name -match "^(Microsoft\.WindowsAppSDK\.[a-zA-Z]+)\.([0-9].*)$" -or
+            $_.Name -match "^(Microsoft\.Windows\.SDK\.BuildTools\.MSIX)\.([0-9].*)$" -or
+            $_.Name -match "^(Microsoft\.Windows\.SDK\.BuildTools)\.([0-9].*)$" -or
```

Comment:
Now that we do BuildTools, we may need to have the wapproj replacement below also use `nugetPackageToVersionTable` instead of just using it for vcxproj and packages.config.